tests/e2e/api/staking.test.ts

```diff
@@ -20,11 +20,12 @@ describe("staking api", () => {
     expect(numDelegators).toEqual(numDelegatorsData[0].num_active_delegator);
   });
 
-  test("it throws if the poolAddress does not exist", async () => {
+  test.only("it throws if the poolAddress does not exist", async () => {
```

Comment:
what does `only` do?

Reply:
ohh it should be here. `only` makes it that only this test will run when you run this test suite